### ISSUE TITLE
Is the Go client actually built?

### ISSUE THREAD

**Issue by pselden** [figroc/tensorflow-serving-client#6]
I don't see the output published anywhere.

**Comment by pselden** [comment]
Eek sorry, should have looked further: https://github.com/figroc/tensorflow-serving-client/archive/v2.1.0.tar.gz

**Comment by figroc** [comment]
Since v2.0, releasing branches contain all the generated source code and are tagged on. I primarily work on the java version. Other versions are briefly tested. All feedback is welcome. ;-)

**Comment by pselden** [comment]
Hey thanks for the response! I know you mentioned you just use Java so don't worry about if you don't know offhand. But do you happen to know why when I try to import it (or go get it), it fails with the following?

```
go get "github.com/figroc/tensorflow-serving-client/go"

go: finding github.com/figroc/tensorflow-serving-client/go latest
go get github.com/figroc/tensorflow-serving-client/go: module github.com/figroc/tensorflow-servin
g-client@upgrade found (v2.1.0+incompatible), but does not contain package github.com/figroc/tensorflow-serving-client/go
```

Do you happen to have any examples of it being used in Go?

**Comment by pselden** [comment]
Also when i tried to download it by git hash I got the following:

```
go get github.com/figroc/tensorflow-serving-client/go@e81d120a6943121a031b9c60aada07c0bc82e634
go: finding github.com/figroc e81d120a6943121a031b9c60aada07c0bc82e634
go: finding github.com e81d120a6943121a031b9c60aada07c0bc82e634
go: finding github.com/figroc/tensorflow-serving-client/go e81d120a6943121a031b9c60aada07c0bc82e634
go: finding github.com/figroc/tensorflow-serving-client e81d120a6943121a031b9c60aada07c0bc82e634
go get: github.com/figroc/tensorflow-serving-client/go@v0.0.0-20200206093350-e81d120a6943 requires
        tensorflow@v0.0.0-00010101000000-000000000000: malformed module path "tensorflow": missing dot in first path element
```

**Comment by figroc** [comment]
It turns out that the tag has to be '_go/vX.X.X_'. There are subpackages _tensorflow_ and _tensorflow\_serving_ which need to be handled properly too. I might go for the vendoring feature.

**Comment by figroc** [comment]
I've managed a version for go (9f2f2b6), and updated r2.0 and r2.1 with new tags v2.0.1 and v2.1.1.

**Comment by pselden** [comment]
I think it's really close, but it doesn't quite work with go 1.13 due to the new SemVer requirements for modules with versions >2.

```
go get github.com/figroc/tensorflow-serving-client@v2.1.1
go: finding github.com/figroc/tensorflow-serving-client v2.1.1
go: finding github.com/figroc v2.1.1
go: finding github.com v2.1.1
go: finding github.com/figroc/tensorflow-serving-client v2.1.1
go get github.com/figroc/tensorflow-serving-client@v2.1.1: github.com/figroc/tensorflow-serving-client@v2.1.1: invalid version: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```

I got it working locally by adding /v2 to the module name everywhere. If you're ok with having that change, I can open a PR.

**Comment by figroc** [comment]
Ah, you are right. No wonder I got the `incompatible` when I `go get` the package. There should be major version tag in the module path. PR is welcome ;-)

**Comment by figroc** [comment]
I've updated r2.0 and r2.1 according to #7.

**Comment by figroc** [comment]
Closed by #7 merged
